### ISSUE TITLE
[Intel HD, Windows 10] Glitchy gray patches on some objects

### ISSUE THREAD

**Issue by kryptot7** [supertuxkart/stk-code#2976]
![fcd79b65-8294-4f31-97d9-33194ae0432b](https://user-images.githubusercontent.com/30781325/32138974-5ade77f8-bc0b-11e7-9686-9608ea206937.gif)
These odd gray patches also show up and flicker on trees and other vegetation.

**Comment by deveee** [comment]
Is this in current git version?

I had similar issue:
https://github.com/supertuxkart/stk-code/issues/2961
the comment with video:
https://youtu.be/np49kRdlkPQ
and modifications in skinning shaders helped.

**Comment by kryptot7** [comment]
No, it's in 0.9.3-rc1. The game runs fine, it just has this graphical glitch.

**Comment by deveee** [comment]
So maybe it's just a case that I can't reproduce it anymore.

Could you post your stdout.log file? It can be found in %appdata%\supertuxkart\0.8.2

It seems to be related to hardware skinning, at least it happened for me after some modifications in HW skinning. You can check it by adding this line to graphical_restrictions.xml:
```
<card os="windows" disable="HardwareSkinning"/>
```
and see if it helped.

And if not, then no idea atm...

**Comment by kryptot7** [comment]
Sure!
[stdout.log](https://github.com/supertuxkart/stk-code/files/1424680/stdout.log)
Where's graphical_restrictions.xml?

**Comment by deveee** [comment]
It's in STK data directory. It will be c:\program files\supertuxkart\data or something like this.

**Comment by deveee** [comment]
Did you already try to disable it in graphical restrictions?

**Comment by kryptot7** [comment]
I added the line, and the issue is still there.

**Comment by deveee** [comment]
One more test... Could you check if it works properly in STK 0.9.2? So that we will see if we introduced a bug somewhere, or maybe it's something wrong in graphics drivers.

I was making more tests and for me it was related only to hardware skinning shaders (precisely to a texelFetch inside a branch).

**Comment by qwertychouskie** [comment]
No response since 2017, this should probably be closed.

**Comment by Alayan-stk-2** [comment]
Most likely an issue with old crappy Intel iGPU drivers.
